jwst/extract_1d/extract.py

```diff
@@ -11,6 +11,7 @@
 from ..assign_wcs import niriss         # for specifying spectral order number
 from ..assign_wcs.util import wcs_bbox_from_shape
 from ..lib import pipe_utils
+from ..master_background.expand_to_2d import get_wavelengths
```

Comment:
At some point (after we deliver B7.3) we really need to move ``get_wavelengths`` to ``lib``, rather than always having to import from ``master_background``. But this is fine for now.